scss/bitstyles/base/forms/settings-select.scss

```diff
@@ -25,9 +25,16 @@ $bitstyles-select-border-active: 2px solid palette('gray', '80') !default;
 $bitstyles-select-background-color-active: transparent !default;
 $bitstyles-select-background-image-active: url("data:image/svg+xml,%3Csvg width='20' height='20' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 100 100'%3E%3Cpath fill='#{$bitstyles-select-color-active}' d='M52.8,68.9,86.32,36a4,4,0,0,0-2.8-6.86h-67A4,4,0,0,0,13.68,36L47.2,68.9A4,4,0,0,0,52.8,68.9Z'/%3E%3C/svg%3E") !default;
 
+//
+// Invalid styles ////////////////////////////////////////
+$bitstyles-select-color-invalid: palette('warning') !default;
+$bitstyles-select-border-invalid: 2px solid palette('warning') !default;
+$bitstyles-select-background-color-invalid: palette('white') !default;
+$bitstyles-select-background-image-invalid: url("data:image/svg+xml,%3Csvg width='20' height='20' xmlns='http://www.w3.org/2000/svg' viewBox='0 0 100 100'%3E%3Cpath fill='#{$bitstyles-select-color-invalid}' d='M52.8,68.9,86.32,36a4,4,0,0,0-2.8-6.86h-67A4,4,0,0,0,13.68,36L47.2,68.9A4,4,0,0,0,52.8,68.9Z'/%3E%3C/svg%3E") !default;
+
 //
 // Disabled styles ////////////////////////////////////////
-$bitstyles-select-color-disabled: palette('black', '40') !default;
-$bitstyles-select-border-disabled: 2px solid !default;
-$bitstyles-select-background-color-disabled: transparent !default;
+$bitstyles-select-color-disabled: palette('text') !default;
+$bitstyles-select-border-disabled: 2px solid palette('black', '20') !default;
+$bitstyles-select-background-color-disabled: palette('black', '20') !default;
```

Comment:
Just to double check: it's on purpose, that both, border and content have the exact same color, right?

<img width="1015" alt="Screenshot 2021-03-10 at 16 17 41" src="https://user-images.githubusercontent.com/3491815/110651888-259fda80-81bc-11eb-9d10-2bea33db61e5.png">


Plus, it looks like the color is not applied correctly to the 🔽 SVG 👀

Reply:
Yes the color is intended to make it fade into the background, instead of having high contrast 🤔 What would you prefer here — some slightly-darker border?

I’ll check the svg color, thanks!

Reply:
> Yes the color is intended to make it fade into the background, instead of having high contrast 🤔 What would you prefer here — some slightly-darker border?

I don't know. It just stood out in this list of select inputs.

It's consistent with the other disabled inputs in Bitstyles, so all fine. Maybe if the color of the 🔽 was adjusted, it would less inviting to still click and interact 🤔

Reply:
I fixed the arrow color, and made the backgrounds of disabled inputs just slightly paler than the borders — you’re right, it looks less clickable. Thanks for the feedback!

Looks like:

<img width="1019" alt="Screenshot 2021-03-11 at 09 22 23" src="https://user-images.githubusercontent.com/2479422/110757318-89bdaf80-824b-11eb-9522-9dc76f646157.png">
<img width="1019" alt="Screenshot 2021-03-11 at 09 22 29" src="https://user-images.githubusercontent.com/2479422/110757324-8b877300-824b-11eb-925e-fe58b4b8af58.png">

Reply:
Gold 🧜‍♂️